src/NUnitFramework/tests/Attributes/TestOrderAttributeTests.cs

```diff
@@ -1,11 +1,9 @@
-﻿using System;
-using NUnit.Framework.Internal;
+﻿using NUnit.Framework.Internal;
 using NUnit.Framework.Internal.Execution;
 using NUnit.TestData;
-using NUnit.TestData.TestCaseSourceAttributeFixture;
 using NUnit.TestUtilities;
 
-namespace NUnit.Framework.Tests.Attributes
+namespace NUnit.Framework.Attributes
```

Comment:
Most attribute tests are in NUnit.Framework.Attributes, with a few in NUnit.Framework.Tests.Attributes. I considered a mass change but didn't want to complicate this PR any further.